microsoft/testsuites/dpdk/dpdksuite.py

```diff
@@ -27,6 +27,7 @@ class Dpdk(TestSuite):
             This test case checks DPDK can be built and installed correctly.
         """,
         requirement=simple_requirement(
+            min_nic_count=2,
```

Comment:
Why does it need at least two nic?

Reply:
https://docs.microsoft.com/en-us/azure/virtual-network/setup-dpdk#prerequisites
![image](https://user-images.githubusercontent.com/10083705/132147301-32423d14-f17f-486c-b80e-1b97e19a063a.png)

Reply:
add comments to explain it. The preconditions should be a part of the description.

Reply:
fixed